wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -275,33 +314,29 @@ static function save_gallery() {
 					$captions[$id] = sanitize_text_field( $image['caption'] );
 				}
 			}
-		}
+			else {
+				self::json_error( 'invalid-input' );
```

Comment:
Should this break out of the for loop on error? Looks like right now it continues if theres an error, but ignores the image that has the error